MaterialsLib/Adsorption/Adsorption.h

```diff
@@ -0,0 +1,107 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef ADSORPTION_H
+#define ADSORPTION_H
+
+#include <array>
+#include <cmath>
+
+#include "Reaction.h"
+
+namespace Adsorption
+{
+
+const double GAS_CONST = 8.3144621;
+
+const double M_N2  = 0.028;
+const double M_H2O = 0.018;
+
+class AdsorptionReaction : public Reaction
+{
+public:
+    // TODO [CL] move those three methods to water properties class
+    static double get_evaporation_enthalpy(const double T_Ads);
```

Comment:
The method names of the class doesn't follow our convention for method names. Is there a special reason?